CK2Plus_expanded/history/titles/d_smolensk.txt

```diff
@@ -46,9 +46,6 @@ holder=626
 1068.9.14={
 	holder = 650
 }
-1169.1.2 = {		#When Kiev lost too much legitimacy to be a proper suzerain of the Rus principalities
-	clear_tribute_suzerain = k_ruthenia
-}
```

Comment:
Did you mean to just delete this one? It looks like you meant to move it but I don't see where it was re-inserted.

Reply:
It's duplicated below.

Reply:
Ah alright then